sqlglot/dialects/oracle.py

```diff
@@ -113,3 +113,6 @@ class Tokenizer(tokens.Tokenizer):
             "VARCHAR2": TokenType.VARCHAR,
             "NVARCHAR2": TokenType.NVARCHAR,
         }
+
+        SINGLE_TOKENS = tokens.Tokenizer.SINGLE_TOKENS.copy()
+        SINGLE_TOKENS.pop("$")
```

Comment:
Moving the dollar sign to the base tokenizer caused a problem in the `V$SESSION` oracle test, because it parsed it as `V AS $SESSION`.